src/lexedata/exporter/cognates.py

```diff
@@ -353,17 +355,22 @@ def get_segments(self, form):
     )
     parser.add_argument(
         "--metadata",
-        help="Path to metadata file for dataset input",
+        nargs="?",
```

Comment:
Oh, by the way, a switch (`--...`) doesn't need an `nargs="?"`, that is assumed.